server/datastore/mysql/microsoft_mdm.go

```diff
@@ -576,3 +576,42 @@ WHERE
 		Detail: dest.Detail,
 	}, nil
 }
+
+func (ds *Datastore) GetMDMWindowsProfile(ctx context.Context, profileUUID string) (*fleet.MDMWindowsConfigProfile, error) {
```

Comment:
Tests for those Datastore methods will be added in a subsequent PR, when the DELETE and GET endpoints will get completed and fully tested.